src/vs/workbench/contrib/notebook/browser/view/renderers/cellOutput.ts

```diff
@@ -852,3 +853,37 @@ const JUPYTER_RENDERER_MIMETYPES = [
 	'application/vnd.jupyter.widget-view+json',
 	'application/vnd.code.notebook.error'
 ];
+
+registerAction2(class extends Action2 {
```

Comment:
we can extend from https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/notebook/browser/contrib/coreActions.ts#L213 and let's move actions out of `view` part.